tests/tests_unit/test_approval_modules_snapshots/cdf_oid_example_data.yaml

```
@@ -39,7 +39,7 @@ Datapoints:
   - pi_163582
   - pi_163657
   - pi_191092
-  dataframe: 75111115425812004176806621799699857630636183615055067181457655101926656558550
+  dataframe: 83920408757402904477028297651743373327423865323707460346684895807103795112712
```

This is a mistake that is causing your tests to fail
```suggestion
  dataframe: 75111115425812004176806621799699857630636183615055067181457655101926656558550
```